### ISSUE TITLE
Not all Errors are returned

### ISSUE THREAD

**Issue by MattBGP** [gookit/validate#231]
**System (please complete the following information):**

 - OS: `windows` [e.g. linux, macOS]
 - GO Version: `1.20`
 - Pkg Version: `v1.5.1`

**Describe the bug**

When validating a Struct, the Errors variable inside of the Validation struct does only hold one error even when multiple errors are inside of the struct to validate.

**To Reproduce**

```go
type ValidationError struct {
	Field  string
	Values map[string]string
}

func main() {
	type MyStruct struct {
		Field1 string `validate:"required"`
		Field2 int    `validate:"required"`
		Field3 string `validate:"required"`
	}

	data := MyStruct{
		Field1: "",
		Field2: 0,
		Field3: "",
	}

	v := validate.Struct(data)

	if !v.Validate() {
		var valErrs []ValidationError
		for field, errs := range v.Errors {
			valErrs = append(valErrs, ValidationError{
				Field:  field,
				Values: errs,
			})
		}
		fmt.Printf("Validation Errors: %+v\n", valErrs)
	} else {
		fmt.Println("No validation errors.")
	}
}
```

Output: `Validation Errors: [{Field:Field1 Values:map[required:Field1 is required to not be empty]}]`

**Expected behavior**

All Validation Errors within the struct should be returned.

**Comment by MattBGP** [comment]
If this wasn't a feature, why does the .All() Method even exist then?

**Comment by MattBGP** [comment]
This has been resolved thanks to #95 

Maybe don't stopping by default would be an option.

**Comment by inhere** [comment]
:) Stop on error performs better. so the default is true
